deployment/scripts/cluster.maintenance.sh

```diff
@@ -5,7 +5,7 @@ SCRIPT_PATH=$(realpath $0)
 SCRIPT_DIR=$(dirname $SCRIPT_PATH)
 
 # check CONFIGURATION
-if [ -z ${CONFIGURATION} ]; then
+if [[ -z "$CONFIGURATION" ]] || [[ $CONFIGURATION == "" ]]; then
```

Comment:
```suggestion
if [[ -z "$CONFIGURATION" ]]; then
```

-z "$CONFIGURATION" should check for empty string, the second condition is implicitly checked by the first.
Tho the change from `-z ${CONFIGURATION}` to `-z "$CONFIGURATION"` introduces this check, I think a recheck if the second condition is needed should be done: https://linuxsimply.com/bash-scripting-tutorial/conditional-statements/if-else/if-string-not-empty/

Reply:
Your are right @ulfgebhardt I checked this and it works in all cirumstances: 'undefined' and ""

Therefore I've overtaken your suggestion in the code at all relevant places with co-authored.